### ISSUE TITLE
urepo down

### ISSUE THREAD

**Issue by sualfred** [zag2me/repository.urepo#1]
Hey mate,

what happened to urepo.org? It's down since 1week<

**Comment by zag2me** [comment]
Yeh took it offline for now, just because so many "builds" were bundling it. The bandwidth got quite immense :) it will be back but maybe at a different URL. 

I also want to create one that is not banned on the main Kodi Forums :)

**Comment by sualfred** [comment]
These damn builds.. I fully understand the reason. 
I like the idea of urepo and the system you created for it. It would be a shame if your work won't be continued somehow.
Is there any way/platform where I can talk/discuss something in private? I don't know if you still have an account at kodi.tv to start a private conversion.

**Comment by zag2me** [comment]
Yes the original idea was to make uploading Add-ons easy and painless for developers.

Sure PM me on kodi forums. @Docwra

**Comment by zag2me** [comment]
Whats your email? your PM's are disabled on kodi forums for some reason.

**Comment by sualfred** [comment]
Even if I start the convo? Good board software. Anyway -> ***@gmail.com

**Comment by mucke5** [comment]
What is the current state?

**Comment by bugatsinho** [comment]
any updates?you could use github as a repository and dont worry about bandwidth!

**Comment by 1024mb** [comment]
Please update with some info.

**Comment by tvaddonsco** [comment]
Get in touch please: webmaster@tvaddons.co

**Comment by TMidi84** [comment]
has anyone been able to hear from Zag about this addon?

**Comment by TMidi84** [comment]
I have found what appears to be all the addon files, as part of a script, but i have no idea how to put those in to KODI

**Comment by zag2me** [comment]
If you have the files, then you can just install form the zip files into Kodi. Its pretty easy.

URepo hosted some great 3rd party Addons not found anywhere else, so let me know how you get on.

**Comment by TMidi84** [comment]
Hey zag2me, did not actually expect a response, but thanks to the surprise.  So here is what I have found: this link points to the zip file.https://v1.kodi-addons.club/addon/repository.urepo/ and this one points to Rob Webset's script for "Suitability" https://github.com/robwebset/script.suitability.  and this last last one https://github.com/zag2me/repository.urepo/raw/master/repository.urepo.zip. They all seem to point to Urepo even though it supposedly offers the ability to just install the zip.  Do I just download the first or last file and then install from zip file?How would I handle the install from repository part?

**Comment by zag2me** [comment]
You can find most of the original Add-ons on Robs repository here (https://github.com/robwebset) but I don't think he supports them any more, they should work though. Just download the zips and install in Kodi via the "Install with zip" feature. Very easy.

**Comment by TMidi84** [comment]
Hey Zag2me, I was able to get this zip into Kodi so a thousand thank you's. 

Are you still working on this add-on at all?

Prior to hearing back from you I reached out to the Kodi Dev team about get some help with bring this add-on to life. Here is the current conversation https://forum.kodi.tv/showthread.php?tid=356505&pid=2971219#pid2971219. This looks promising.

**Comment by TMidi84** [comment]
Hey Zag2me, I was able to get this zip into Kodi so a thousand thank you's. 

Are you still working on this add-on at all?

Prior to hearing back from you I reached out to the Kodi Dev team about get some help with bring this add-on to life. Here is the current conversation https://forum.kodi.tv/showthread.php?tid=356505&pid=2971219#pid2971219. This looks promising.
